content/guide/avalanche9000-motivation.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: What is the Motivation behind the Avalanche9000 Upgrade?
+description: Learn about the largest network upgrade of Avalanche, that will lower the barrier to launch a blockchain and enable permissionless vlaidation of L1s.
+date: 2024-08-15
+authors: [martin_eckardt, owenwahlgren]
+topics: [Avalanche]
+---
+
+Avalanche9000 will be the largest network upgrade of Avalanche to date. This upgrade will introduce a new concept called L1s, which will replace the current Subnet model. In this article we will discuss the current limitations of Subnets, and how the changes suggested in [ACP-77](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md) will address these limitations in the Etna Upgrade.
+
+<YouTube id="eczkjrGG_As" />
+^^ placeholder
+
+## ACP-77
+
+### How Subnets Work Today
+
+![Subnets](/common-images/multi-chain-architecture/acp-77/subnets.png)
+
+The Avalanche network has a large set of validators who all participate in running and validating the Primary Network.
+
+The Primary Network consists of the C-Chain, P-Chain, and X-Chain, and is where most of the transactions on Avalanche are processed.
+
+In order to scale horizontally and offer more blockspace, a subset of validators can opt-in to validate an additional blockchain in parallel with the Primary Network, we call this a Subnet.
+
+### Subnets -> L1s
+With ACP-77 Subnets will become their own Layer 1 blockchain.
+
+![L1s](/common-images/multi-chain-architecture/acp-77/l1s.png)
+
+Instead of using subsets of validators from the Primary Network - we now use a new distinct set of validators.
```

Comment:
```suggestion
Instead of using a subset of Primary Network Validators, new L1s can now use a distinct set of validators.
```
Avoid the use of "we" "us" "you" and other pronouns.